pkg/disk/partition.go

```diff
@@ -87,7 +87,7 @@ func (p *Partition) IsBIOSBoot() bool {
 		return false
 	}
 
-	return p.Type == BIOSBootPartitionGUID
+	return p.Type == BIOSBootPartitionGUID || p.Type == DosBIOSBootID
```

Comment:
Let's add this for now but maybe add a note that we should probably drop BIOS boot from DOS PTs.  From what I understand, it's not needed and the `DosBIOSBootID` I added was wrong.

Reply:
Yeah, after a bit of reading up on this I came to the same conclusion, AIUI BIOS boot is not needed on dot PTs, there the MBR gap is used to store the second stage of grub.